### ISSUE TITLE
Question about code

### ISSUE THREAD

**Issue by TuRan-sino** [hitz-zentroa/GoLLIE#16]
This work is a very impressive, but i have some question about your code.
In the line 158 - 171 of src/dataset/dataset.py 
Can these lines of code be transformed into formula below

![image](https://github.com/hitz-zentroa/GoLLIE/assets/51948125/841ef09a-710b-4b39-a05d-75f5df11885f)

I don't understand the purpose of this formula

**Comment by ikergarcia1996** [comment]
Hi @TuRan-sino 

The code allows to set of a percentage of the total loss for the prompt tokens and the result tokens (the tokens that the model must generate) during training. If you set the parameter `prompt_loss_weight: 0.05`, the prompt tokens will contribute 5% to the total loss, while the result tokens will contribute 95%. This formula calculates the weight that should be assigned to each prompt token so that they account for 5% of the total loss.

We conducted some tests with this parameter during the development of GoLLIE and found that setting the weight of the loss tokens to 0 yields the best results. That's why every configuration in [GoLLIE/configs](https://github.com/hitz-zentroa/GoLLIE/tree/main/configs) has `prompt_loss_weight` set to `0`. Therefore, the result of this formula will always be 0. However, we have retained this parameter in the code for future experiments.
